tests/accessibility/getAccessibilityTree.test.ts

```diff
@@ -58,24 +56,23 @@ test(
         ↳ description: "extended description"
       text "extended description"
     `);
-    expect(await getAccessibilityTree(body, { includeText: true }))
```

Comment:
Since we changed the default value of includeText to true, this snapshot and the one above are the same. I changed the `includeText` property here so we test both cases.